plugin/helpers_plugin.go

```diff
@@ -54,3 +54,16 @@ func (p *HelpersImpl) ensureServerVersion(required string) error {
 	}
 	return nil
 }
+
+// GetPluginAssetURL builds a URL to the resource from asset directory.
+func (p *HelpersImpl) GetPluginAssetURL(pluginID, asset string) (*url.URL, error) {
+	if len(pluginID) == 0 || len(asset) == 0 {
+		return nil, errors.New("empty pluginID or directory name provided")
```

Comment:
Let's break these checks apart into discrete errors to make it clear to the plugin author what went wrong. Also, I think it would be an empty `asset name` not `directory name`?

Reply:
@lieut-data Cool! Thanks for the feedback, I will work on it. I think it'll be better to split the errors for different scenarios.